### PR TITLE
Fix some (Britain-)english names of reinforced blocks

### PR DESCRIPTION
**Pull request by RedstoneDubstep**
Well, as you may or may not know, I am a perfectionist regarding the names of reinforced blocks being equal to the names of their vanilla variants. This pull request started with the Ingot blocks or the former Blocks of Ingot, and then I've found some other minor, minor issues. If there are any problems, please use comments, reviews etc. Thanks!

### CODE REVIEW

File: src/main/resources/assets/securitycraft/lang/en_us.json [hunk 1]
```diff
@@ -97,14 +97,14 @@
 	"block.securitycraft.reinforced_dirt": "Reinforced Dirt",
 	"block.securitycraft.reinforced_cobblestone": "Reinforced Cobblestone",
 	"block.securitycraft.reinforced_sandstone": "Reinforced Sandstone",
-	"block.securitycraft.reinforced_chiseled_sandstone": "Reinforced Chiseled Sandstone",
+	"block.securitycraft.reinforced_chiseled_sandstone": "Reinforced Chiselled Sandstone",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/en_us.json]: Revert, English (US) spells it with one l.

File: src/main/resources/assets/securitycraft/lang/en_us.json [hunk 2]
```diff
@@ -113,14 +113,14 @@
 	"block.securitycraft.reinforced_stone_bricks": "Reinforced Stone Bricks",
 	"block.securitycraft.reinforced_mossy_stone_bricks": "Reinforced Mossy Stone Bricks",
 	"block.securitycraft.reinforced_cracked_stone_bricks": "Reinforced Cracked Stone Bricks",
-	"block.securitycraft.reinforced_chiseled_stone_bricks": "Reinforced Chiseled Stone Bricks",
-	"block.securitycraft.reinforced_stone_brick_slab": "Reinforced Stone Bricks Slab",
+	"block.securitycraft.reinforced_chiseled_stone_bricks": "Reinforced Chiselled Stone Bricks",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/en_us.json]: Revert, English (US) spells it with one l.

File: src/main/resources/assets/securitycraft/lang/en_us.json [hunk 3]
```diff
@@ -146,12 +146,12 @@
...
 	"block.securitycraft.reinforced_acacia_log": "Reinforced Acacia Log",
 	"block.securitycraft.reinforced_dark_oak_log": "Reinforced Dark Oak Log",
-	"block.securitycraft.reinforced_gold_block": "Reinforced Block of Gold",
-	"block.securitycraft.reinforced_iron_block": "Reinforced Block of Iron",
-	"block.securitycraft.reinforced_diamond_block": "Reinforced Block of Diamond",
-	"block.securitycraft.reinforced_emerald_block": "Reinforced Block of Emerald",
+	"block.securitycraft.reinforced_gold_block": "Reinforced Gold Block",
+	"block.securitycraft.reinforced_iron_block": "Reinforced Iron Block",
+	"block.securitycraft.reinforced_diamond_block": "Reinforced Diamond Block",
+	"block.securitycraft.reinforced_emerald_block": "Reinforced Emerald Block",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/en_us.json]: Revert, English (US) spells it with `of`.

File: src/main/resources/assets/securitycraft/lang/en_us.json [hunk 4]
```diff
@@ -168,16 +168,16 @@
 	"block.securitycraft.reinforced_green_wool": "Reinforced Green Wool",
 	"block.securitycraft.reinforced_red_wool": "Reinforced Red Wool",
 	"block.securitycraft.reinforced_black_wool": "Reinforced Black Wool",
-	"block.securitycraft.reinforced_quartz_block": "Reinforced Block of Quartz",
-	"block.securitycraft.reinforced_chiseled_quartz_block": "Reinforced Chiseled Quartz Block",
+	"block.securitycraft.reinforced_quartz_block": "Reinforced Quartz Block",
+	"block.securitycraft.reinforced_chiseled_quartz_block": "Reinforced Chiselled Quartz Block",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/en_us.json]: Revert, English (US) uses `of` and spells the other with one l.

File: src/main/resources/assets/securitycraft/lang/en_us.json [hunk 5]
```diff
@@ -168,16 +168,16 @@
...
 	"block.securitycraft.reinforced_quartz_pillar": "Reinforced Quartz Pillar",
 	"block.securitycraft.reinforced_quartz_stairs": "Reinforced Quartz Stairs",
 	"block.securitycraft.reinforced_quartz_slab": "Reinforced Quartz Slab",
 	"block.securitycraft.reinforced_prismarine": "Reinforced Prismarine",
 	"block.securitycraft.reinforced_prismarine_bricks": "Reinforced Prismarine Bricks",
 	"block.securitycraft.reinforced_dark_prismarine": "Reinforced Dark Prismarine",
 	"block.securitycraft.reinforced_red_sandstone": "Reinforced Red Sandstone",
-	"block.securitycraft.reinforced_chiseled_red_sandstone": "Reinforced Chiseled Red Sandstone",
+	"block.securitycraft.reinforced_chiseled_red_sandstone": "Reinforced Chiselled Red Sandstone",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/en_us.json]: Revert, English (US) spells it with one l.

File: src/main/resources/assets/securitycraft/lang/en_us.json [hunk 6]
```diff
@@ -285,11 +285,11 @@
 	"block.securitycraft.reinforced_diorite_slab": "Reinforced Diorite Slab",
 	"block.securitycraft.reinforced_normal_stone_slab": "Reinforced Stone Slab",
 	"block.securitycraft.trophy_system": "Trophy System",
-	"block.securitycraft.crystal_quartz": "Block of Crystal Quartz",
-	"block.securitycraft.chiseled_crystal_quartz": "Chiseled Crystal Quartz Block",
+	"block.securitycraft.crystal_quartz": "Crystal Quartz Block",
+	"block.securitycraft.chiseled_crystal_quartz": "Chiselled Crystal Quartz Block",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/en_us.json]: Revert, English (US)'s quartz uses of and the other is spelled with one l.

File: src/main/resources/assets/securitycraft/lang/en_us.json [hunk 7]
```diff
@@ -285,11 +285,11 @@
...
 	"block.securitycraft.crystal_quartz_pillar": "Crystal Quartz Pillar",
-	"block.securitycraft.reinforced_crystal_quartz_block": "Reinforced Block of Crystal Quartz",
-	"block.securitycraft.reinforced_chiseled_crystal_quartz_block": "Reinforced Chiseled Crystal Quartz Block",
+	"block.securitycraft.reinforced_crystal_quartz_block": "Reinforced Crystal Quartz Block",
+	"block.securitycraft.reinforced_chiseled_crystal_quartz_block": "Reinforced Chiselled Crystal Quartz Block",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/en_us.json]: Revert, English (US) uses of and spells the other with one l.

File: src/main/resources/assets/securitycraft/lang/en_us.json [hunk 8]
```diff
@@ -306,7 +306,7 @@
 	"block.securitycraft.reinforced_dark_oak_pressure_plate": "Reinforced Dark Oak Pressure Plate",
 	"block.securitycraft.reinforced_red_sand": "Reinforced Red Sand",
 	"block.securitycraft.reinforced_bookshelf": "Reinforced Bookshelf",
-	"block.securitycraft.reinforced_redstone_block": "Reinforced Block of Redstone",
+	"block.securitycraft.reinforced_redstone_block": "Reinforced Redstone Block",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/en_us.json]: Revert, English (US) spells it with of.

File: src/main/resources/assets/securitycraft/lang/en_us.json [hunk 9]
```diff
@@ -404,7 +404,7 @@
...
+	"help.securitycraft.block_pocket_manager.info": "The Block Pocket Manager is used to manage the Block Pocket, which is a room that only its owner and whitelisted players can enter. When opening its GUI, you see a selection of sizes. Choose one, then build the Block Pocket around the manager. Make sure that the manager is placed on the outside of the bottom layer. The Block Pocket is a cube of the size you chose. The corners are built with Reinforced Chiselled Crystal Quartz, the blocks between the corners are built with correctly oriented Reinforced Crystal Quartz Pillars, and the floor/wall/ceiling is built with Block Pocket Walls. See this image for an example of how it looks: https://i.imgur.com/opc7dqO.png - Clicking \"Activate\" after the structure is built will activate any added modules. See the customization GUI for more information (rightclick with Universal Block Modifier).",
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/en_us.json]: Revert, English (US)'s quartz is spelled with one l.

**Comment by RedstoneDubstep** [comment]
> Some of the lines in en_GB are incorrectly indented. Please fix this.
sorry for my english skills, what does "intended" mean in this context?
If it means "ausgerichtet" like not all of the lines are at the same position, then I'll fix this, if not, please tell me what I should fix then. Thanks!

File: src/main/resources/assets/securitycraft/lang/en_gb.json [hunk 10]
```diff
@@ -3,12 +3,31 @@
...
+        "block.securitycraft.reinforced_chiseled_crystal_quartz_block": "Reinforced Chiselled Crystal Quartz Block",
+        "block.securitycraft.reinforced_redstone_block": "Reinforced Redstone Block",
+        
+        "help.securitycraft.block_pocket_manager.info": "The Block Pocket Manager is used to manage the Block Pocket, which is a room that only its owner and whitelisted players can enter. When opening its GUI, you see a selection of sizes. Choose one, then build the Block Pocket around the manager. Make sure that the manager is placed on the outside of the bottom layer. The Block Pocket is a cube of the size you chose. The corners are built with Reinforced Chiselled Crystal Quartz, the blocks between the corners are built with correctly oriented Reinforced Crystal Quartz Pillars, and the floor/wall/ceiling is built with Block Pocket Walls. See this image for an example of how it looks: https://i.imgur.com/opc7dqO.png - Clicking \"Activate\" after the structure is built will activate any added modules. See the customization GUI for more information (rightclick with Universal Block Modifier)."
+
+
```
Comment (bl4ckscor3) [inline src/main/resources/assets/securitycraft/lang/en_gb.json]: Missed these two empty lines in my last review, sorry about that. Once they're removed, I'll merge!

Comment (RedstoneDubstep) [reply src/main/resources/assets/securitycraft/lang/en_gb.json]: I have absolutely no clue how the empty lines came there, but now they're gone.

**Pull request merged**